docs/user/glossary.rst

```diff
@@ -39,36 +44,97 @@ so that you have a reference for how we're using them.
       and rules for publication of documentation can be :doc:`automated </automation-rules>`.
       Similar to :term:`Docs as Code`.
 
+   pinning
+      *Pinning* software requirements is the most important technique to make a project :term:`reproducible`.
+      To *pin* a requirement means to specify which version that should be used.
```

Comment:
This sentence should probably be the first one, since it explains what _pinning is_ exactly. The rest is just context for better understanding of this sentence, IMO.